.buildkite/gen-pipeline.sh

```diff
@@ -94,23 +92,28 @@ run_mpi_pytest() {
   local test=$1
   local queue=$2
 
-  local exclude_keras_if_needed=""
+  local exclude_keras=""
   if [[ ${test} == *"tf2_"* ]] || [[ ${test} == *"tfhead"* ]]; then
     # TODO: support for Keras + TF 2.0 and TF-Keras 2.0
-    exclude_keras_if_needed="| sed 's/test_keras.py//g' | sed 's/test_tensorflow_keras.py//g'"
+    exclude_keras="| sed 's/test_keras.py//g' | sed 's/test_tensorflow_keras.py//g'"
   else
-    exclude_keras_if_needed="| sed 's/[a-z_]*tensorflow2[a-z_.]*//g'"
+    exclude_keras="| sed 's/[a-z_]*tensorflow2[a-z_.]*//g'"
   fi
 
-  local exclude_interactiverun="| sed 's/test_interactiverun.py//g' | sed 's/test_spark_keras.py//g' | sed 's/test_spark_torch.py//g'"
+  local exclude_elastic=""
+  if [[ ${test} == *"py2_"* ]]; then
```

Comment:
Yes, the `Barrier` feature used in the driver does not exist in Python 2.  So rather than finding a less elegant solution, I thought it would be better to just drop Python 2 support :).